### PR TITLE
Convert from bash script to Home Assistant integration

### PR DESCRIPTION
**Pull request by firstof9**
Convert from using bash script (advanced linux experience needed), to Home Assistant integration (easier user experience).
Reorganized repo for HACS usage.

### CODE REVIEW

File: custom_components/lock-manager/const.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+DOMAIN = "lock-manager"
+VERSION = "0.0.4"
+ISSUE_URL = "https://github.com/firstof9/lock-manager"
```
Comment (firstof9) [inline custom_components/lock-manager/const.py:3]: We'll need to update this.

File: custom_components/lock-manager/manifest.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "lock-manager",
+  "name": "Lock Manager",
+  "documentation": "https://github.com/firstof9/lock-manager",
```
Comment (firstof9) [inline custom_components/lock-manager/manifest.json:4]: This will need updating as well.

File: custom_components/lock-manager/manifest.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "dependencies": [],
+  "codeowners": ["@firstof9"],
```
Comment (firstof9) [inline custom_components/lock-manager/manifest.json:6]: As well as this

**Comment by FutureTense** [comment]
Approved.  Nice job!

**Comment by firstof9** [comment]
Oops closed instead of merged?

**Comment by FutureTense** [comment]
That was it.  I’m of the opinion that git sucks.  Mercurial is better

**Pull request merged**